pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -0,0 +1,371 @@
+package coinbase
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+type channelType struct {
+	Name       string   `json:"name"`
+	ProductIDs []string `json:"product_ids,omitempty"`
+}
+
+func (c channelType) String() string {
+	return fmt.Sprintf(`(Name:       %s,
```

Comment:
Looks like you add a `tab` before the value?
![image](https://github.com/user-attachments/assets/34337ce1-c3be-46aa-8207-3fa3ae58ca48)

You can insert a tab using the `\t` directive, but in this case, I guess it's either a typo or you intend to align each line manually. However, when reading logs on different platforms such as Kibana, they may parse the logs differently, which can affect the alignment and make it look different from what you intended.

Reply:
Got it.
I will simplify the format string.